transform/macros/map_class_fp.sql

```diff
@@ -0,0 +1,17 @@
+{% macro map_class_fp(class_fp, k, v) -%}
+
+{% set class_fp_dict = {
+    "M2" : "A military or other defense installation entirely within a place",
+    "C1" : "An active incorporated place that does not serve as a county subdivision equivalent",
+    "U1" : "A census designated place with an official federally recognized name",
+    "U2" : "A census designated place without an official federally recognized name"
```

Comment:
This macro is an interesting pattern as an alternative to a look-up table. I kind of like it, as it essentially moves look up tables into code. I'd be curious of @AeriShan-ODI has any thoughts on this.

Reply:
When I was approaching this look up tables came up as an option, but I didn't like the idea of the "clutter" (not sure why I think of it that way, is data hoarding a thing?) of an additional table! I didn't consider fully how this could be a pattern that moves it into code so thank you for recognizing that!

Reply:
Yeah, another option is to use dbt "seeds", which might be a bit more idiomatic. I don't have a ton of experience with either approach.